src/fsharp/vs/ServiceFormatting/CodeFormatter.fs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information.
```

Comment:
This line is hilarious ;-)

Reply:
I've just copied this from another file.

Reply:
Done. I removed the MS copyright and added the following comment:

```fsharp
// Copied from https://github.com/dungpa/fantomas and modified by Vasily Kirichenko
```

I'm not sure I should add my name though.